src/gen_dragonfruit/gd_dev_apogee_sp215.cpp

```diff
@@ -1,27 +1,41 @@
 #include "gd_dev_apogee_sp215.h"
 
 void gd_dev_apogee_sp215_open(void){
-    Wire.begin(9600);
+
+    /* Initiate Wire library and join I2C bus as slave */
+    Wire.begin();
+
+    /* Begin transmission to the I2C slave device with the given address */
+    Wire.beginTransmission(_DEV_ADDR_GD_APOGEE_SP215_);
+
+    /* Enable read bit and select 16-bit sample rate and select sampling rate
+    of 8 SPS. */
+    Wire.write(0x8C);
+
+    Wire.endTransmission();
 }
 
-int gd_dev_apogee_sp215_read(void){
-    int value = 555;
+uint32_t gd_dev_apogee_sp215_read(void){
+    uint32_t value = 555;
     #ifndef SEN_STUB
-    Wire.beginTransmission(_PIN_GD_APOGEE_SP215_);
 
-    /* Options */
-    Wire.write(0x8C);
+    /* Begin transmission to the I2C slave device with the given address */
+    Wire.beginTransmission(_DEV_ADDR_GD_APOGEE_SP215_);
```

Comment:
The name `_DEV_ADDR_GD_APOGEE_SP215_` is misleading since the APOGEE_SP215 sensor does not actually have an address.